src/org/openstreetmap/josm/tools/Utils.java

```diff
@@ -1663,11 +1659,7 @@ public static int clamp(int val, int min, int max) {
             throw new IllegalArgumentException(MessageFormat.format("Parameter min ({0}) cannot be greater than max ({1})", min, max));
         } else if (val < min) {
             return min;
-        } else if (val > max) {
-            return max;
-        } else {
-            return val;
-        }
+        } else return Math.min(val, max);
```

Comment:
See above comment.

Reply:
I'm going to revert this as well, see above.

Reply:
Reverted in 8eed35f8.